earth_enterprise/src/fusion/gecombineterrain/combineterrain.cpp

```diff
@@ -533,9 +542,12 @@ void TerrainCombiner::PacketWriteThread() {
     PacketInfo* packet = NULL;
     while(PopWriteQueue(&packet)) {
       if (packet != NULL) {
+        BEGIN_IO_LOGGING(pwThread,"CombineTerrain_PacketWrite","Packet write thread");
         WritePacket(packet);
         delete packet;
         packets_processed++;
+        END_IO_LOGGING(pwThread);
+        // how to handle exceptions?
```

Comment:
What do you need to handle during exceptions?  Is there something special we need to log about an exception?
  Currently, a block logger will have its destructor called when control flows out of the block because of an exception, which would calculate elapsed timing, etc.

Reply:
When originally going through, I had comments scattered as notes. Forgot to remove this one.